packages/@glimmer/compiler/index.ts

```diff
@@ -7,8 +7,9 @@ export {
   s,
   unicode,
 } from './lib/builder/builder';
-export { Builder, type BuilderStatement } from './lib/builder/builder-interface';
+export { type BuilderStatement } from './lib/builder/builder-interface';
 export { defaultId, precompile, precompileJSON, type PrecompileOptions } from './lib/compiler';
+export * from '@glimmer/constants';
```

Comment:
How many things are in here?
Why *?

(This is an expansion of public api for glimmer compiler, so i just want to make sure this makes sense, rather than mostly a convenience (because idk))

Reply:
This is a good point. There are a few things that were re-exported before when they were a single enum *type* that I carried forward and shouldn't have.